### PR TITLE
Attention and feedforward dropout

### PR DESCRIPTION
**Pull request by eonglints**
AudioLM and Text-Free Prosody-Aware Generative Spoken Language Modeling (PGSLM) use a dropout of 0.1 (PGSLM explicitly states this is for both attention and feedforward layers, AudioLM isn't so specific).

Also a minor fix to enable the `get_tokens` function added in [0.0.62](https://github.com/lucidrains/audiolm-pytorch/commit/dfd4b8030c64370a80972c36f6d2b8d1b9c263f4)

### CODE REVIEW

**Comment by lucidrains** [comment]
very nice! i will also add https://github.com/lucidrains/x-transformers#forgetful-causal-mask to all three transformers later this week (and perhaps even dropout of tokens in cross attention)

https://arxiv.org/abs/2211.11586 all the research is lining up!

**Comment by eonglints** [comment]
Ooh, that forgetful causal mask looks very interesting... I wonder whether it would allow ["fill-in-the-middle"](https://arxiv.org/abs/2207.14255) type inference or if it's just limited to training...? Will have a read of the paper. 

And the token drop-out paper results are pretty amazing! Hadn't seen that one, thanks!
